test/ganache-driver.ts

```diff
@@ -0,0 +1,45 @@
+import { spawn, ChildProcess } from 'child_process';
+import { ETHEREUM_URL } from '../eth';
+import fetch from 'node-fetch';
+import { retry } from 'ts-retry-promise';
+
+export const ganacheDriver = {
+    process: null as ChildProcess | null,
+    async startGanache() {
+        if (ganacheDriver.process) {
+            throw new Error(`ganache-cli process already running! PID=${ganacheDriver.process.pid}`);
+        }
+        const process = spawn(
+            'ganache-cli',
+            [
+                '-p',
+                '7545',
+                '-i',
+                '5777',
+                '-a',
+                '100',
+                '-m',
+                'vanish junk genuine web seminar cook absurd royal ability series taste method identify elevator liquid'
+            ],
+            { stdio: 'pipe' }
+        );
+        ganacheDriver.process = process;
+        await retry(
```

Comment:
Should this kill the ganache process if this fails?

Reply:
added error handling